### PR TITLE
Add use cases section

### LINKED ISSUE #24: Add use-cases and requirements section
Would be nice to add use-cases section, I have few in mind:

- Metal detector web app
- Providing input using magnet button for VR enclosures (e.g., vrbutton.appspot.com)
- User-specific manual fusion with other low-level sensors

### PR DESCRIPTION
**Pull request by riju**
Add use cases for magnetometer.
This is an intial list of use cases.
We can add more use cases later.

Fixes #24

***
[Preview](https://s3.amazonaws.com/pr-preview/riju/magnetometer/addUseCases.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/magnetometer/4484e03...riju:88e2dd4.html)

### CODE REVIEW

**Comment by riju** [comment]
@alexshalamov , @anssiko : PTAL.

File: index.bs [hunk 1]
```diff
@@ -31,6 +30,42 @@ urlPrefix: https://w3c.github.io/sensors; spec: GENERIC-SENSOR
     text: construct a sensor object; url: construct-sensor-object
 </pre>
 
+
+<pre class=biblio>
```
Comment (alexshalamov) [inline index.bs]: Could you also inline biblio.json? If I remember correctly, online generator doesn't work with external biblio files.

Comment (riju) [inline index.bs]: Thanks, done :)

File: index.bs [hunk 2]
```diff
@@ -156,6 +196,27 @@ In addition to the above spatial variations of the <a>magnetic field</a>, time b
...
+Magnetometers can be used for a variety of use-cases, such as:
+
+- Metal detector web application.
+- Providing input using magnetic button for VR enclosures [[VRBUTTON]].
+- Gesture recognition based on magnetic field. Various interactions like writing, signing and playing an instrument can also be enabled using a magnet like a rod, pen or a ring [[MAGITACT]].
+  The user makes coarse gestures in the 3D space around the device using the magnet. Movement of the magnet affects the magnetic field sensed by the compass sensor integrated in the device.
+  The temporal pattern of the gesture is then used as a basis for sending different interaction commands to the mobile device. Zooming, turning pages, accepting/rejecting calls, clicking items
+  are some of the use cases.
+- In cases, where it is not possible to use [=orientation-sensor=], user may need to fuse magnetometer data with with other low-level sensor data, to calculate orientation.
+- Indoor Navigation System may use magnetometer on mobile phones [[MAGINDOORPOS]]. Specific use cases for indoor navigation may include Proximity Advertising, Way-finding in malls or airports Geo-Fencing, etc.
```
Comment (anssiko) [inline index.bs]: "Indoor Navigation System", "Proximity Advertising", "Wayfinding", "Geo-Fencing" should be lowercase.

**Comment by pozdnyakov** [comment]
lgtm, assumed nits from Anssi are resolved.

File: index.bs (hunk 2 above)
Comment (riju) [inline index.bs]: Done :)

**Pull request merged**